Cargo.lock

```diff
@@ -1906,6 +1906,7 @@ dependencies = [
  "fedimint-client",
  "fedimint-core",
  "fedimint-ln-common",
+ "fedimint-logging",
```

Comment:
How did this happen?! There was no `Cargo.toml` change :eyes:

Reply:
It might be in a different commit. I did change Cargo.tomls and there was a rebase conflict at some point on `Cargo.lock`.